<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateTerrain</class>
 <widget class="QDialog" name="GenerateTerrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Terrain</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Number of octaves</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="comboBoxOctaves"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Noise frequency</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBoxFrequency"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Tile Size</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboBoxTileSize"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="comboBoxSize"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButtonGenerate">
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
